server/market/market.go

```diff
@@ -599,21 +602,35 @@ func (m *Market) Cancelable(oid order.OrderID) bool {
 // means: (1) an order in the book or epoch queue, (2) type limit with
 // time-in-force standing (implied for book orders), and (3) AccountID field
 // matching the provided account ID.
-func (m *Market) CancelableBy(oid order.OrderID, aid account.AccountID) bool {
+func (m *Market) CancelableBy(oid order.OrderID, aid account.AccountID) (bool, error) {
```

Comment:
I made this change to troubleshoot, but I think it might be useful. Technically, could just return the `error` without the `bool`, but I'm not crazy about the semantics there.